src/kvirc/kernel/KviIrcConnection.h

```diff
@@ -160,8 +161,8 @@ class KVIRC_API KviIrcConnection : public QObject
 
 	KviIrcConnectionStateData * m_pStateData; // owned, never null
 
-	KviPointerList<KviChannelWindow> * m_pChannelList; // owned, never null, elements shallow
-	KviPointerList<KviQueryWindow> * m_pQueryList;     // owned, never null, elements shallow
+	std::vector<KviChannelWindow *> m_pChannelList; // owned, never null, elements shallow
+	std::vector<KviQueryWindow *> m_pQueryList;     // owned, never null, elements shallow
```

Comment:
This looks wrong...
Either the comment is outdated, and it's not owned anymore, or the windows are owned, then they should be stored as `vector<unique_ptr<KQW>>`